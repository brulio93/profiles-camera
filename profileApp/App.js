'use strict';
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import TabNav from './src/routers/Router';
import ProfileStore from './src/stores/ProfileStore';
import ImportStore from './src/stores/ImportStore';
import {Provider} from 'mobx-react';

export default class App extends React.Component {
  render() {
    const stores = {ProfileStore, ImportStore};
    return (
      <Provider {...stores}>
        <TabNav />        
      </Provider>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
