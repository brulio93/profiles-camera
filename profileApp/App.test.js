import React from 'react';
import 'react-native';
import App from './App';
import renderer from 'react-test-renderer';
import Enzyme from "enzyme";

const mockCamera = require('./src/mocks/Camera').default;
const ProfileStore = require('./src/stores/ProfileStore').default;
jest.mock('react-native-camera', () => mockCamera);

it('renders without crashing', () => {
  const wrapper = renderer.create(<App />);
  expect(wrapper).toMatchSnapshot();
});