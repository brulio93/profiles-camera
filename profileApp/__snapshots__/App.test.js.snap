// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  loaded={
    Array [
      0,
    ]
  }
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      Object {
        "flex": 1,
      },
    ]
  }
>
  <View
    collapsable={undefined}
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Object {
        "alignItems": "stretch",
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
      testID={undefined}
    >
      <View
        style={
          Object {
            "flex": 1,
            "overflow": "hidden",
          }
        }
      >
        <View
          onLayout={[Function]}
          style={
            Array [
              Object {
                "flex": 1,
              },
              undefined,
            ]
          }
        >
          <View
            onMoveShouldSetResponder={[Function]}
            onMoveShouldSetResponderCapture={[Function]}
            onResponderEnd={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderReject={[Function]}
            onResponderRelease={[Function]}
            onResponderStart={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            onStartShouldSetResponderCapture={[Function]}
            style={
              Array [
                Object {
                  "flex": 1,
                  "flexDirection": "column-reverse",
                },
                Object {
                  "backgroundColor": "#000",
                },
              ]
            }
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                collapsable={undefined}
                pointerEvents="auto"
                style={
                  Object {
                    "backgroundColor": "#E9E9EF",
                    "bottom": 0,
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "shadowColor": "black",
                    "shadowOffset": Object {
                      "height": 0,
                      "width": 0,
                    },
                    "shadowOpacity": 0.2,
                    "shadowRadius": 5,
                    "top": 0,
                    "transform": Array [
                      Object {
                        "translateX": 0,
                      },
                      Object {
                        "translateY": 0,
                      },
                    ],
                  }
                }
              >
                <RCTScrollView>
                  <View />
                </RCTScrollView>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      style={null}
      testID={undefined}
    />
    <View
      style={null}
      testID={undefined}
    />
  </View>
  <View
    collapsable={undefined}
    style={
      Object {
        "backgroundColor": "#33691E",
        "borderTopColor": "rgba(0, 0, 0, .3)",
        "borderTopWidth": 0.5,
        "flexDirection": "row",
        "height": 49,
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "flex": 1,
          "justifyContent": "flex-end",
        }
      }
      testID={undefined}
    >
      <View
        style={
          Object {
            "flexGrow": 1,
          }
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        collapsable={undefined}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "rgba(52, 120, 246, 1)",
            "fontSize": 10,
            "marginBottom": 1.5,
            "marginLeft": 0,
            "marginTop": 0,
            "textAlign": "center",
          }
        }
      >
        List
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "flex": 1,
          "justifyContent": "flex-end",
        }
      }
      testID={undefined}
    >
      <View
        style={
          Object {
            "flexGrow": 1,
          }
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        collapsable={undefined}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "rgba(146, 146, 146, 1)",
            "fontSize": 10,
            "marginBottom": 1.5,
            "marginLeft": 0,
            "marginTop": 0,
            "textAlign": "center",
          }
        }
      >
        Contact
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "flex": 1,
          "justifyContent": "flex-end",
        }
      }
      testID={undefined}
    >
      <View
        style={
          Object {
            "flexGrow": 1,
          }
        }
      >
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
        <View
          collapsable={undefined}
          style={
            Object {
              "alignItems": "center",
              "bottom": 0,
              "justifyContent": "center",
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        collapsable={undefined}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "rgba(146, 146, 146, 1)",
            "fontSize": 10,
            "marginBottom": 1.5,
            "marginLeft": 0,
            "marginTop": 0,
            "textAlign": "center",
          }
        }
      >
        Import
      </Text>
    </View>
  </View>
</View>
`;
